azure-webapp-gradle-plugin/src/main/resources/schema/GradleWebAppConfiguration.json

```diff
@@ -210,6 +250,48 @@
           "certificate"
         ]
       }
+    },
+    "deployment-resource": {
+      "type": "object",
+      "title": "Deployment Resource",
+      "description": "Deployment Resource Configuration for Maven plugin for Azure Web App",
+      "properties": {
+        "type": {
+          "type": "string",
+          "pattern": "(?i)^(war|jar|ear|lib|script|static|startup|zip)$"
+        },
+        "directory": {
+          "$ref": "#/definitions/non-empty-string"
+        },
+        "targetPath": {
+          "$ref": "#/definitions/non-empty-string"
+        },
+        "includes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/non-empty-string"
+          }
+        },
+        "excludes": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/non-empty-string"
+          }
+        }
+      }
+    },
+    "deployment": {
```

Comment:
Do we support customize artifact to deploy in gradle?

Reply:
No

Reply:
This part is designed based on Maven Resource, so may be removed in gradle